hr_cae_documents/security/ir.model.access.csv

```diff
@@ -0,0 +1,3 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_ir_attachment_category_user,access_ir_attachment_category_user,model_ir_attachment_category,base.group_user,1,0,0,0
```

Comment:
I'm quite surprised to see that users can't delete attachements 😉

Reply:
They ccan't delete attachement **category**!